### DISCUSSION TITLE
2 Quick Newbie Questions

### DISCUSSION THREAD

**Discussion by mettaseva** [oleeskild/obsidian-digital-garden#102] [Q&A] [answered]
- Is there a way to unpublish files without deleting them from the Vault?
- Where is the publish flag supposed to display?

Thanks in advance,
Metta

**Comment by oleeskild** [comment, marked as answer]
1. Yes. Simply remove the "dg-publish: true" attribute from the note you want to delete. Then use the "Open Publication Center" command and click the "Delete notes from garden" button. 
2. This is part of the [frontmatter](https://help.obsidian.md/Advanced+topics/YAML+front+matter) of the note. So at the very top of your note it should look something like this:
<img width="839" alt="CleanShot 2022-11-13 at 17 09 01@2x" src="https://user-images.githubusercontent.com/6201338/201531794-da140b39-f9d3-414f-9e7a-8a996fdf64cc.png">

So firstly three dashes, ---, then a new line with dg-publish: true, and then a new line with three dashed, --- 

```
---
dg-publish: true
---
```

**Reply by mettaseva** [reply]
Thanks so much @oleeskild! 
Appreciate your prompt and helpful feedback.
